sycl/doc/extensions/IntelGPU/IntelGPUDeviceInfo.md

```diff
@@ -0,0 +1,113 @@
+# SYCL(TM) Proposal: New device descriptors for Intel GPUs
+
+**IMPORTANT**: This specification is a draft.
+
+**NOTE**: Khronos(R) is a registered trademark and SYCL(TM) is a trademark of the Khronos Group, Inc.
+
+New device descriptors will be added to provide access to low-level hardware details about Intel GPU devices.  This information will be useful to developers tuning specifically for those devices.
+
+This proposal details what is required to provide this information as a SYCL extensions.
+
+## Feature Test Macro ##
+
+The same Feature Test Macro will be used for the new device descriptor support.  It will be defined as:
+
+    #define SYCL_EXT_INTEL_GPU_DEVICE_INFO 1
+
+
+
+# Intel GPU PCI Address #
+
+A new device descriptor will be added which will provide the PCI address of an Intel GPU in BDF format.  BDF format contains the address as: `domain:bus:device.function`.
+
+This support can only be provided when using the Level Zero backend \(BE\).  This is the default for Intel GPUs.  The OpenCL BE does not provide the PCI address at this time.
+
+
+## Device Information Descriptors ##
+
+| Device Descriptors | Return Type | Description |
+| ------------------ | ----------- | ----------- |
+| info\:\:device\:\:ext\_intel\_gpu\_pci\_address | std\:\:string | For Level Zero BE, returns the PCI address in BDF format: `domain:bus:device.function`.|
+
+
+## Aspects ##
+
+A new aspect, ext\_intel\_gpu\_pci\_address, will be added.
+
+## Error Condition ##
+
+The function device\:\:get_info\(\) will return an empty string if the device does not support aspect\:\:ext\_intel\_gpu\_pci\_address.
+
+
+## Example Usage ##
+
+The PCI address can be obtained using the standard get\_info() interface.
+
+    if (dev.has(aspect::ext_intel_gpu_pci_address) {
```

Comment:
Nit: Missing closing paren here and in the other examples.

Reply:
thanks!